orc8r/gateway/python/magma/common/service_registry.py

```diff
@@ -78,6 +78,23 @@ def list_services():
         """
         return ServiceRegistry.get_registry()["services"]
 
+    @staticmethod
```

Comment:
Looks like the linter wants @classmethod instead.. https://wemake-python-stylegui.de/en/latest/pages/usage/violations/oop.html#wemake_python_styleguide.violations.oop.StaticMethodViolation

I don't use Python often enough to have an opinion on this, will research best practices :)

Reply:
The ServiceRegistry class is full of @staticmethod... The linter is complaining about a lot of other things. Still working on pleasing it.

Reply:
Yeah dw about existing lint warnings. Would be good to not introduce new warnings OR identify any linter warnings that should be disabled.

Reply:
Fixed all reported issues. I don't know why Python Format check is still failing. It's not telling me either...

Reply:
It looks there are some leftover formatting changes for `lte/gateway/python/magma/kernsnoopd/handlers.py`
```
Running formatting commands for lte/gateway/python/magma/kernsnoopd/__init__.py...
Running formatting commands for lte/gateway/python/magma/kernsnoopd/handlers.py...
Rewriting lte/gateway/python/magma/kernsnoopd/handlers.py
```

What are you running locally?

Reply:
I run `autopep8` directly and it's not making any changes to the existing file. I couldn't use the precommit.py script as it complained about not finding `magma/py-lint:latest` and not being able to download the image because of docker login issues.

Reply:
Did you follow the instructions here? https://docs.magmacore.org/docs/next/lte/dev_unit_testing#format-agw

You have to use the build flag first to build the image

Reply:
I hadn't followed this one. I was looking at the [contributing conventions](https://magma.github.io/magma/docs/next/contributing/contribute_conventions#python). Have made the required changes now.

Reply:
i should update that file then :)